dev/run

```diff
@@ -249,9 +249,15 @@ def hack_default_ini(ctx, node, content):
     coffeejs = os.path.join(ctx['rootdir'], "share", "server", "main-coffee.js")
 
     repl = "javascript = %s %s" % (couchjs, mainjs)
+    if os.name == 'nt':
+        # Erlang always wants UNIX-style paths
+        repl = repl.replace("\\", "/")
```

Comment:
Sorry, Ilya, it does not:

```python
>>> foo = "E:\\this\\is\\a\\test"
>>> foo
'E:\\this\\is\\a\\test'
>>> foo[2]
'\\'
>>> os.path.normcase(foo)
'e:\\this\\is\\a\\test'
```
We always want backslashes to be forward slashes for these specific paths because Erlang is too stupid to use the correct path separator character on Windows, it _always_ wants the UNIX separator. I'm not sure of any other way to do this. Besides, there are other places in this script where we already use the same `string.replace` syntax. Unless you want me to convert all of them... :wink: